app/assets/javascripts/star_rating.js

```diff
@@ -0,0 +1,15 @@
+$(document).ready(function(){
```

Comment:
ok. this is good jQuery, but is there a reason you decided to go in this direction as opposed to a ReactJS component?